cookbooks/bcpc/attributes/default.rb

```diff
@@ -261,11 +254,6 @@
 default['bcpc']['member_role'] = 'Member'
 default['bcpc']['admin_email'] = 'admin@localhost.com'
```

Comment:
`admin_email` is used by our SSL certificate recipe but I have tried to normalize some of our wonky e-mail handling in a new update.